ul: add page (#3282)
tldr-pages/tldr: pages/linux/ul.md

# ul

> Performs the underlining of a text.

> > To underline a text is necessary to underline each character separately.

Reviewer: What precisely do you mean by this?

Author: If you want to underline the character H you can write something like ``_\bH`` in a file or using ``echo`` and call ``ul``. However, if you want to underline Hello, you can surround the whole word with underline ``_\bHello_\b``, you need to do this for each character ``_\bH_\be_\bl_\bl_\bo``.

Reviewer: Ah, I see. Perhaps we could reword this to make it clearer. What about this?
```suggestion
> Each character in a given string must be underlined separately.
```
